tutorials/fundamentals/YMD.md

```diff
@@ -0,0 +1,24 @@
+To format a date to YYYY MM DD in JavaScript, you can use the `toLocaleDateString()` function in combination with the `split()`, `reverse()`, and `join()` functions.
+You can take advantage of the European style date format, which puts day first, as opposed to the US format which puts month first.
+
+```javascript
+const date = new Date();
+
+date.toLocaleDateString('en-GB').split('/').reverse().join(''); // 20211124
+```
+
+## Using String Math
+
+A more interesting approach is to use string math, which concatenates strings.
```

Comment:
"A more readable approach is to convert the date to a number and then convert the number to a string."

The downside of the above approach is that it's a bit magic - I would imagine most people outside of the UK don't know off the top of their head that UK dates are DD MM YYYY, this approach is easier to understand at a glance.